scripts/in_container/run_prepare_provider_packages.sh

```diff
@@ -63,7 +72,6 @@ if [[ -z "$*" ]]; then
         echo
         exit 1
     fi
-    PROVIDER_PACKAGES=$(python3 "${PREPARE_PROVIDER_PACKAGES_PY}" list-backportable-packages)
```

Comment:
This was the problem -- we were resetting this list to the backport list, even when building normal packages.